src/iiiftilesource.js

```diff
@@ -181,20 +182,34 @@ $.extend( $.IIIFTileSource.prototype, $.TileSource.prototype, /** @lends OpenSea
     },
 
     /**
-     * Return the tileSize for the given level.
+     * Return the tileWidth for the given level.
      * @function
      * @param {Number} level
-    */
+     */
+    getTileWidth: function( level ) {
```

Comment:
Yeah, I was tring not to break anything, not being completely sure why certain decisions were made inside of the constructor of IIIFTileSource. I think I can manage updating the old logic that uses either tile_width or tile_height as tileSize. A demo IIIF server to test again would be helpful.